gravitee-am-ui/src/app/domain/applications/application/advanced/password-policy/password-policy.component.html

```diff
@@ -20,6 +20,15 @@
     <div fxFlex="70">
       <form (ngSubmit)="update()" (keydown.enter)="false" fxLayout="column" #applicationForm="ngForm">
         <div *ngIf="editMode">
+
+          <div fxLayout="column" style="margin-top: 10px;">
+            <mat-slide-toggle [checked]="inherited" (change)="setInheritConfigurationValue($event);formChange()">
+              Inherit configuration
+            </mat-slide-toggle>
+            <mat-hint style="font-size: 75%;">Enable passwordless authentication (WebAuthn / FIDO).</mat-hint>
```

Comment:
Inherit Password policy settings from the security domain.

Reply:
ok